tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "tslint-config-standard",
+  "rules": {
+    "trailing-comma": [true, {"multiline": "always", "singleline": "never"}]
```

Comment:
Yeah, this is testing the idea that we can do "mostly standard" not sure it's a good door to open.  Kinda trying it on to see if we scream running, and easy enough to remove.